packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -2344,11 +2344,12 @@ export class _BinaryWriter {
      */
     private _resizeBuffer(byteLength: number): ArrayBuffer {
         const newBuffer = new ArrayBuffer(byteLength);
-        const oldUint8Array = new Uint8Array(this._arrayBuffer);
+        let oldUint8Array = new Uint8Array(this._arrayBuffer);
         const newUint8Array = new Uint8Array(newBuffer);
-        for (let i = 0, length = newUint8Array.byteLength; i < length; ++i) {
-            newUint8Array[i] = oldUint8Array[i];
+        if (oldUint8Array.length > newUint8Array.length) {
+            oldUint8Array = oldUint8Array.subarray(0, newUint8Array.length);

```

Comment:
could this not be merged in line 2347 directly in the constructor parameter ?

Reply:
yes, updated